apps/dashboard/app/pages/invite/accept.vue

```diff
@@ -1,3 +1,54 @@
+<script setup lang="ts">
+import { useMutation, useQuery } from "@tanstack/vue-query";
+import { acceptInviteInfoByToken, getInviteInfoByToken } from "~/api/invites";
+import { queryKeys } from "~/api/queryKeys";
+
+definePageMeta({
+  auth: "guest",
+  layout: "auth",
+});
+
+const authStore = useAuthStore();
+const router = useRouter();
+const query = useRoute().query;
+const token = computed(() => query.token?.toString() ?? "");
+
+const { data, isPending } = useQuery({
+  queryKey: queryKeys.invites.token(token.value),
+  async queryFn() {
+    return await getInviteInfoByToken(token.value);
+  },
+});
+
+const { mutate, isPending: isMutatePending } = useMutation({
+  async mutationFn() {
+    const newMember = await acceptInviteInfoByToken(token.value);
+    await authStore.fetchUser();
+    authStore.switchOrg(newMember.orgId);
+    router.push("/");
+  },
+});
+</script>
+
 <template>
-  <p>Accept an invite from email</p>
+  <Form
+    v-if="data"
+    @submit="mutate()"
+  >
+    <AuthHeading>You've been invited</AuthHeading>
+    <div class="border-neutral-500 mb-4 border rounded-xl p-3">
+      <Bold>{{ data?.org.name }}</Bold>
+      <p>{{ data?.org.description }}</p>
```

Comment:
```suggestion
      <Bold>{{ data.org.name }}</Bold>
      <p>{{ data.org.description }}</p>
```

These shouldn't be needed since there's a v-if=data :)